survey/src/main/resources/data.sql

```diff
@@ -0,0 +1,11 @@
+INSERT INTO survey(id, name, schema_json)
+SELECT
+    gen_random_uuid(),
```

Comment:
For current stage is better to use the const id in fixture (allowing frontend to get this form just without guesssing  id)